resume/src/App.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
```

Comment:
Great use of components, I like how you built out every section of the resume into a separate component! Makes it very easy to add on other sections in the future